src/pieces/assets/assets_command.py

```diff
@@ -5,7 +5,19 @@
 from .assets_api import AssetsCommandsApi
 from pieces.gui import show_error,print_model_details,space_below,double_line,deprecated
 from pieces.settings import Settings
-
+import subprocess
+from prompt_toolkit import PromptSession
+from prompt_toolkit.document import Document
+from prompt_toolkit.key_binding import KeyBindings
+from pygments import highlight
+from pygments.lexers import get_lexer_by_name, guess_lexer
+from pygments.formatters import TerminalFormatter
+import json
+import tempfile
+
+#Ensure the data_path directory exists
+os.makedirs(Settings.pieces_data_dir, exist_ok=True)
```

Comment:
Would like to remove all config related command to a separate file in the commands directory

Reply:
made this change by creating a separate config_command.py file in commit [c172f20](https://github.com/pieces-app/cli-agent/pull/152/commits/c172f20c58c3fd1b686fdaae1350a4a6f8b06f34) and then calling refactoring the asset_command.py accordingly in commits [4b7d5e8](https://github.com/pieces-app/cli-agent/pull/152/commits/4b7d5e827dbe9e381872a9b52b26bd3f17bd4105) and [11836a7](https://github.com/pieces-app/cli-agent/pull/152/commits/11836a7c8680831db98339ecd013f5860d1244e6)

Reply:
Don't forget to change the app.py also to point to the config function

Reply:
added change in [3b897c0](https://github.com/pieces-app/cli-agent/pull/152/commits/3b897c0e34773860d5c91f630b4c3bdc91a8c26e) commit